python/perspective/setup.py

```diff
@@ -311,6 +311,7 @@ def run_check(self):
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
         "Programming Language :: Python :: 3.11",
+        "Framework :: Jupyter :: JupyterLab :: Extensions :: Prebuilt",
```

Comment:
Here are 2 others you might consider adding, though discoverability via this mechanism is not terribly popular https://github.com/timkpaine/ipydagred3/blob/0d8f858b84b7f65d9de5aa5ae59e54a502e3b5b2/pyproject.toml#L30

Reply:
https://jupyterlab.readthedocs.io/en/stable/extension/extension_dev.html#pypi-trove-classifiers

Reply:
Yep that's where I got mine from